testnet/obscuro-node-status.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+#
+# This script displays the Obscuro node health status
+#
+
+help_and_exit() {
+    echo ""
+    echo "Usage: "
+    echo "      -  $(basename "${0}") "
+    echo ""
+    exit 1  # Exit with error explicitly
+}
+# Ensure any fail is loud and explicit
+set -euo pipefail
+
+# Define local usage vars
+start_path="$(cd "$(dirname "${0}")" && pwd)"
+testnet_path="${start_path}"
+
+# Fetch options
+for argument in "$@"
+do
+    key=$(echo $argument | cut -f1 -d=)
+    value=$(echo $argument | cut -f2 -d=)
+
+    case "$key" in
+
+            --help)                     help_and_exit ;;
+            *)
+    esac
+done
+
+
+net_status=$(curl -s --request POST 'http://127.0.0.1:13000' \
```

Comment:
Does this assume you always run the node on port 13000? I thought in the start script the port could be set differently

Reply:
Good point, I'll hook these up to the flags 👌

Reply:
Actually, just checked and we hardcode those ports in the startup script. I guess there's no point in changing them until it's strictly necessary ? WDYT ?

Reply:
When you start the node you give the p2p_address i.e. "51.189.31.34:1000" ... is 1000 not the port the node listens on?

Reply:
aha got it